test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/InjectDirectiveTest.cs

```diff
@@ -194,9 +164,10 @@ private RazorEngine CreateEngine()
         {
             return RazorEngine.Create(b =>
             {
-                // Notice we're not registering the InjectDirective.Pass here so we can run it on demand.
```

Comment:
This change is needed because if we don't register the early on, the `DirectiveRemovalIRPass` removes all directive nodes before this pass gets a chance.